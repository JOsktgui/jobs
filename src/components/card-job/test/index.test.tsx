import React from 'react';
import renderer from 'react-test-renderer';
import CardJob from '../';

test('render Search component', () => {
  const props = {
    title: 'Jobss',
    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit.',
    locationNames: 'Lima, Perú',
    applyUrl: 'https://github.com/JOsktgui/jobs',
    company: {
      name: 'Jobs',
      slug: 'jobs'
    },
    commitment: {
      title: 'Full-Time',
      slug: 'full-time'
    },
    postedAt: '2019-08-12T20:19:52.000Z',
    countries: [
      { name: 'Perú', slug: 'peru' }
    ]
  };

  const component = renderer.create(<CardJob {...props} />);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
