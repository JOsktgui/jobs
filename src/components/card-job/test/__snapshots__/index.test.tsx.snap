// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Search component 1`] = `
<div
  className="container-card-job"
>
  <h2
    className="container-card-job__title"
  >
    Jobss
  </h2>
  <div
    className="container-card-job__item"
  >
    <span>
      Ubicación:
    </span>
     
    Lima, Perú
  </div>
  <div
    className="container-card-job__item"
  >
    <span>
      Empresa:
    </span>
     
    Jobs
  </div>
  <div
    className="container-card-job__item"
  >
    <span>
      Modalidad:
    </span>
     
    Full-Time
  </div>
  <div
    className="container-card-job__item"
  >
    <span>
      Publicado:
    </span>
     
    5 meses
  </div>
  <div
    className="container-card-job__item"
  >
    <span>
      Países:
    </span>
     
    Perú
  </div>
  <p
    className="container-card-job__description"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
  </p>
  <a
    className="container-card-job__apply"
    href="https://github.com/JOsktgui/jobs"
    rel="noopener noreferrer"
    target="_blank"
  >
    APLICAR
  </a>
</div>
`;
